lib/migration.js

```diff
@@ -66,60 +66,6 @@ function mixinMigration(MsSQL) {
     });
   };
 
-  MsSQL.prototype.autoupdate = function(models, cb) {
-    var self = this;
-    if ((!cb) && ('function' === typeof models)) {
-      cb = models;
-      models = undefined;
-    }
-    // First argument is a model name
-    if ('string' === typeof models) {
-      models = [models];
-    }
-
-    models = models || Object.keys(this._models);
-    async.each(models, function(model, done) {
-      if (!(model in self._models)) {
-        return process.nextTick(function() {
-          done(new Error(g.f('Model not found: %s', model)));
-        });
-      }
-      self.showFields(model, function(err, fields) {
-        self.showIndexes(model, function(err, indexes) {
-          if (!err && fields.length) {
-            self.alterTable(model, fields, indexes, done);
-          } else {
-            self.createTable(model, done);
-          }
-        });
-      });
-    }, cb);
-  };
-
-  MsSQL.prototype.isActual = function(cb) {
-    var ok = false;
-    var self = this;
-    async.each(Object.keys(this._models), function(model, done) {
-      self.showFields(model, function(err, fields) {
-        self.showIndexes(model, function(err, indexes) {
-          self.alterTable(model, fields, indexes, function(err, needAlter) {
```

Comment:
If it calls `alterTable` here, we **could not** use the function in base connector.
`getAddModifyColumns` and `getColumnsToDrop` are missing.